api/can_api_v2_definition.py

```diff
@@ -60,6 +60,14 @@ class NonCovidPatientsMethod(enum.Enum):
     ESTIMATED_FROM_TOTAL_ICU_ACTUAL = "estimated_from_total_icu_actual"
 
 
+class Demographics(base_model.APIBaseModel):
```

Comment:
I don't think `Demographics` describes what this holds very clearly. How about calling this DemographicDistribution or Distributions or ByDemographicBucket?